guide/inheritance-wallet/backup.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: guide
+title: Wallet backup
+description: This page gives an overview of how the Joneses backup their inheritance wallet.
+nav_order: 3
+parent: Inheritance wallet
+permalink: /guide/inheritance-wallet/backup/
+main_classes: -no-top-padding
+image_base: /assets/images/guide/inheritance-wallet/
+images_wallet-backup:
+    - file: wallet-backup/app-home-backup-before
+      alt: Image of the app home screen showing a reminder to backup the wallet.
+      caption: The app shows a reminder to save the backup kit.
+    - file: wallet-backup/wallet-settings-backup-kit
+      alt: Screen showing instructions on how to store the wallet backup.
+      caption: The backup page highlights the most important points about the wallet backup and provides a link to learn more.
+    - file: wallet-backup/backup-save-dialog
+      alt: Placeholder screen representing the operating system native file download dialog.
+      caption: The file is saved using the native file download flow.
+    - file: wallet-backup/backup-success
+      alt: Screen showing a success message.
+      caption: The success screen reminds the user once again to not store the backup kit in the same place as the private key backups.
+    - file: wallet-backup/app-home-backup-done
+      alt: Image of the home screen that does not show the backup reminder anymore.
+      caption: The app home screen does not show the backup reminder anymore.
+---
+
+<!--
+
+Editor's notes
+
+This page covers how users backup their wallet and an example approach of how to store the backup material.  
+
+Illustration sources
+
+https://www.figma.com/file/h5GP5v5dYfpXXfEUXf6nvC/Inheritance-wallet?type=design&node-id=6293%3A21917&mode=design&t=I2e3qgqYRGpAGyaQ-1 
+
+-->
+
+# Wallet backup 
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+The inheritance wallet described in this reference design is a multi-key wallet with time-based recovery. When backing up such a wallet, users must back up not only their own private keys but also the wallet configuration, as covered [here](https://bitcoin.design/guide/savings-wallet/time-based-recovery/#wallet-backup).
```

Comment:
Make sure to use relative links instead of absolute ones. Several instances of that on this page.